terraform/environments/example/loadbalancer.tf

```diff
@@ -0,0 +1,103 @@
+# Build loadbalancer security group
+resource "aws_security_group" "example-load-balancer-sg" {
+  name        = "example-lb-sg"
+  description = "controls access to load balancer"
+  vpc_id      = data.aws_vpc.shared.id
+  tags = merge(local.tags,
+    { Name = lower(format("lb-sg-%s-%s-example", local.application_name, local.environment)) }
+  )
+ 
+ # Set up the ingress and egress parts of the security group
+  }
```

Comment:
Could you move this curly brace so it finishes the above block above the comment please